templates/frontend/pages/purchaseIndividualSubscription.tpl

```diff
@@ -15,11 +15,14 @@
 		{csrf}
 
 		<fieldset>
+			<legend>
+				{translate key="payment.subscription.purchase"}
+			</legend>
 			<div class="fields">
 				<div class="subscription_type">
 					<label>
 						<span class="label">
-							{translate key="user.subscriptions.form.typeId"}
+							{translate key="payment.subscription.purchase"}
```

Comment:
I think this should stay as `user.subscriptions.form.typeId`. That would make the legend "Purchase Subscription" and this particular field "Subscription Type".